include/openenclave/internal/crypto/cert.h

```diff
@@ -377,7 +377,8 @@ typedef struct _oe_cert_config
     size_t ext_oid_size;
 } oe_cert_config_t;
 
-#define OE_MAX_CERT_SIZE 8192
+/* includes subject name, issuer name and valid date range */
+#define OE_MIN_CERT_SIZE 256
```

Comment:
Is 256 always sufficient to contain those fields? Would there be a corner case?

Reply:
Although it is large enough for the two examples I've got, I'll discuss it with others in the attestation team.